tutorials/wpa-enterprise-eap-ttls-pap.rst

```diff
@@ -217,7 +217,7 @@ Paste the following JSON in the ``Raw JSON Editing`` field.
     :target: ../_images/config-variable-mac-json.png
 
 You can refer the :doc:`Configuration Variables
-</user/configuration-variables>` section of this documentation for more
+</controller/user/variables>` section of this documentation for more
```

Comment:
this is not the only link that needs to be updated here, please update the link to the page "Configuring FreeRADIUS for WPA Enterprise (EAP-TTLS-PAP)" in the ansible module. 

There's many instances of other links that need to be updated in these pages too, please revise by looking for "<https://github.com/openwisp".